homeassistant/components/lamarzocco/strings.json

```diff
@@ -0,0 +1,72 @@
+{
+  "config": {
+    "flow_title": "La Marzocco Espresso {host}",
+    "abort": {
+      "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "no_machines": "No machines found in account",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "machine_not_found": "The configured machine was not found in your account. Did you login to the correct account?",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "password": "Your password from the La Marzocco app",
+          "username": "Your username from the La Marzocco app"
+        }
+      },
+      "machine_selection": {
+        "data": {
+          "host": "[%key:common::config_flow::data::ip%]",
+          "machine": "Machine"
+        },
+        "data_description": {
+          "host": "Local IP address of the machine"
+        }
+      },
+      "host_selection": {
+        "data": {
+          "host": "[%key:common::config_flow::data::ip%]"
+        },
+        "data_description": {
+          "host": "[%key:component::lamarzocco::config::step::machine_selection::data_description::host%]"
+        }
+      },
+      "reauth_confirm": {
+        "title": "[%key:common::config_flow::title::reauth%]",
+        "description": "One or more of your provided credentials are invalid.",
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      }
+    }
+  },
+  "entity": {
+    "switch": {
+      "main": {
+        "name": "Main"
+      },
+      "auto_on_off": {
+        "name": "Auto on/off"
```

Comment:
Does it turn off automatically or does it turn on automatically?

Reply:
Both, it enables the schedule

Reply:
What is this function called in the app?

Reply:
![image](https://github.com/home-assistant/core/assets/24647999/7af24f9e-9ce2-431b-899e-b0b61bf30189)